rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -40,6 +40,7 @@ nova_db_max_pool_size: 120
 nova_db_pool_timeout: 60
 nova_cpu_allocation_ratio: 2.0
 nova_ram_allocation_ratio: 1.0
+nova_console_type: "spice"
```

Comment:
This is required in the earlier releases as maas can not interpret the role defaults. This is following the already merged pattern in kilo (https://github.com/rcbops/rpc-openstack/blob/kilo/rpcd/etc/openstack_deploy/user_variables.yml#L41)